ansible_base/lib/abstract_models/common.py

```diff
@@ -62,6 +62,11 @@ def save(self, *args, **kwargs):
         '''
         update_fields = list(kwargs.get('update_fields', []))
 
+        # If the updating fields lists matches the objects `not_user_modified_fields` then we can just save and return
+        if update_fields == getattr(self, 'not_user_modified_fields', []):
```

Comment:
:thinking: #308 should make this unnecessary. The field in question is already marked `editable=False`, so that should be enough.